docs/hardware.rst

```diff
@@ -279,35 +279,70 @@ We currently recommend the Intel NUC for SecureDrop servers.
     could include: speakers, cameras, microphones, fingerprint readers,
     wireless, and Bluetooth cards.
 
-Intel NUC
-~~~~~~~~~
+.. _nuc7_recommendation:
+
+Intel 7th-gen NUC
+~~~~~~~~~~~~~~~~~
 
 The Intel NUC (Next Unit of Computing) is an inexpensive, quiet, low-power
 device that can be used for the SecureDrop servers. There are a
-`variety of models <https://www-ssl.intel.com/content/www/us/en/nuc/products-overview.html>`__
-to choose from. We recommend the
-`NUC5i5MYHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc5i5myhe.html>`__
-because it has a mid-range CPU (the 5th generation Intel i5), a Mini
-DisplayPort port for a monitor, and two USB 3.0 ports for faster OS
-installation and data transfer.
+`variety of models <https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html>`__
+to choose from. 
+
+The NUCs typically come as kits, and some assembly is required. You will need to
+purchase the RAM and hard drive separately for each NUC and insert both into the
+NUC before it can be used. We recommend:
+
+-  2x 240GB HDDs (2.5")
+-  1x memory kit of compatible 2x4GB sticks
+   -  You can put one 4GB memory stick in each of the servers.
+
+.. note:: We recommend the use of HDDs over SSDs, as it's more difficult to 
+          securely delete data from SSDs.
```

Comment:
The previous recommendation is to use SSDs. I'm assuming that was performance-motivated? Does the concern re: secure deletion outweigh performance considerations, even in a context like ours, where the most sensitive data is stored only in encrypted form?

Reply:
The impression I got from recent research was that a) wear leveling makes verifiable deletion from SSDs problematic and b) we'll probably be moving to a reduced number of passes for secure deletion anyway, reducing the time for the system's most significant I/O-bound task.

Reply:
In both cases, we can't really provide meaningful mitigations against seizure or forensics without at least full disk encryption in place. Furthermore, modern filesystems (Xenial default partion format is ext4) have journaling functionality, would also make some data available post-deletion unless disabled.

As stated above, HDDs provide better guarantees than SSDs for deleting single files (though SSDs might offer better Secure Deletion of the entire drive). We should also keep in mind that we use the `secure_delete` pragma for the (unencrypted) database as well: https://github.com/freedomofpress/securedrop/blob/develop/securedrop/alembic/versions/faac8092c123_enable_security_pragmas.py#L21

Reply:
...but this is something that needs more discussion so I'll revert the change for now.